test-result-summary-client/src/Build/TestTable.jsx

```diff
@@ -133,6 +134,17 @@ export default class TestTable extends Component {
                             <LinkOutlined />{' '}
                         </Tooltip>{' '}
                     </a>
+                    <Divider type="vertical" />
+                    <a
+                        target="_blank"
+                        href={rerunUrl}
```

Comment:
We should not directly use the rerun link as it is general to rerun the test build. We need to update it for each test. For example https://github.com/adoptium/aqa-test-tools/blob/5dcfaeec709af6ce9f01527f4ce576d4371713d9/test-result-summary-client/src/Build/GitNewIssue.jsx#L53-L56